resources/js/Components/Drafts/NftDraftCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link } from "@inertiajs/react";
+import cn from "classnames";
+import { NftDraftFooter } from "./NftDraftCard.blocks";
+import { GalleryCoverImage } from "@/Components/Galleries/GalleryPage/GalleryCoverImage";
+import { GalleryHeading, GalleryStats, NftImageGrid } from "@/Components/Galleries/NftGalleryCard.blocks";
+
+export const NftDraftCard = ({ gallery }: { gallery: App.Data.Gallery.GalleryData }): JSX.Element => (
```

Comment:
This is still a valid remark:
> Something to consider is that as my understanding the draft cards are not going to come from the database so you probably need to pass a js object instead of reusing the old card component that depends on a database model, we can refactor that on another task once we have more progress with drafts but at least is something to have in mind

Please check this PR: https://github.com/ArdentHQ/dashbrd/pull/315